pkgs/games/sil-q/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib, stdenv, fetchFromGitHub, writeScript, makeWrapper, ncurses, libX11 }:
+
+let
+  version = "v1.5.0";
+  src = fetchFromGitHub {
+    owner = "sil-quirk";
+    repo = "sil-q";
+    rev = version;
+    sha256 = "sha256-v/sWhPWF9cCKD8N0RHpwzChMM1t9G2yrMDmi1cZxdOs=";
+  };
+  setup = writeScript "setup" ''
+    mkdir -p "$ANGBAND_PATH"
+    # Copy all the data files into place.
+    cp -ar $1/* "$ANGBAND_PATH"
+    # The copied files are not writable, make them so.
+    chmod +w -R "$ANGBAND_PATH"
+  '';
+in stdenv.mkDerivation {
+  pname = "sil-q";
+  inherit version src;
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ ncurses libX11 ];
+
+  # Makefile(s) and config are not top-level.
+  sourceRoot = "source/src";
+
+  prePatch = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
Will add! What's the reasoning behind this? I guess this applies to the "original" `default.nix` in `sil` as well then.